### PR TITLE
Wrapping lists addon

### PR DESCRIPTION
**Pull request by TheColaber**
For taking over the Better3.0 Extension

### CODE REVIEW

File: addons/wrap-lists/addon.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Wrap Lists",
+  "description": "todo.",
```
Comment (RedGuy12) [inline addons/wrap-lists/addon.json:3]: then do

File: content-scripts/cs.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ try {
   throw "Scratch Addons: not first party iframe";
 }
 
+console.log(chrome);
+
```
Comment (RedGuy12) [inline content-scripts/cs.js:8]: uhhh

File: manifest.json [hunk 3]
```diff
@@ -47,7 +47,8 @@
     "webRequestBlocking",
     "storage",
     "contextMenus",
-    "alarms"
+    "alarms",
+    "identity"
```
Comment (RedGuy12) [inline manifest.json:51]: ...

File: content-scripts/cs.js (hunk 2 above)
Comment (TheColaber) [reply content-scripts/cs.js:8]: oops

File: addons/wrap-lists/userscript.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    updateListItems();
+    new MutationObserver(updateListItems).observe(list, { attributes: true, attributeFilter: ["style"] });
+  }
+}
+function updateListItems() {
+  for (const value of document.querySelectorAll("[class*='monitor_value-inner']")) {
+    value.style.height = "fit-content";
+    value.style.wordBreak = "break-all";
+    value.style.whiteSpace = "normal";
+  }
```
Comment (WorldLanguages) [inline addons/wrap-lists/userscript.js:21]: Hmm why not do this from CSS?

**Pull request merged**